File: java/problems/p0058_length_of_last_word/Solution.java
package problems.p0058_length_of_last_word;

class Solution {
    public int lengthOfLastWord(String s) {
        String trimmed = s.trim();
        int ret = 0;
        for (int i = trimmed.length() - 1; i >= 0; i--) {
            char c = trimmed.charAt(i);
            if (c == ' ') {
                break;
            }
            ret++;
        }
        return ret;
    }
}

File: typescript/problems/p0058_length_of_last_word/solution.ts
function lengthOfLastWord(s: string): number {
  let trimmed: string = s.trim();
  let ret: number = 0;

  for (let i = trimmed.length - 1; i >= 0; i--) {
    let c: string = trimmed[i];
    if (c === " ") {
      break;
    }
    ret++;
  }
  return ret;
}

export default lengthOfLastWord;
